assets/templates/partials/filter.tmpl

```diff
@@ -137,6 +139,108 @@
     </noscript>
 </form>
 
+{{ if $population_types }}
+<form id="filterForm" method="get" class="js-auto-submit__form">
+    <input type="hidden" name="q" value="{{ $query }}"/>
+    <input type="hidden" name="sort" value="{{$sort}}"/>
+    <fieldset class="ons-fieldset">
```

Comment:
The separation between each collapsible is quite tight - might be better done as an accordion: 

https://service-manual.ons.gov.uk/design-system/components/accordion#example.accordion.html

Reply:
@allandegnan may be better to progress as is and refactor later.